src/pages/difficulty.js

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import { Link } from 'gatsby';
+import '../styles/style.scss';
+
+// data
+const data = [
+    {
+        text: 'Very easy',
+        dc:'5'
+    },
+    {
+        text: 'Easy',
+        dc:'10'
+    },
+    {
+        text: 'Moderate',
+        dc:'15'
+    },
+    {
+        text: 'Hard',
+        dc:'20'
+    },
+    {
+        text: 'Very hard',
+        dc:'25'
+    },
+    {
+        text: 'Nearly impossible',
+        dc:'30'
+    }
+];
+
+// data
+const track = [
+    {
+        text: 'Soft surface such as snow',
+        dc:'10'
+    },
+    {
+        text: 'Dirt or grass',
+        dc:'15'
+    },
+    {
+        text: 'Bare stone',
+        dc:'20'
+    },
+    {
+        text: 'Each day since the creature passed',
+        dc:'(+5)'
+    },
+    {
+        text: 'The creature left a trail, such as blood',
+        dc:'(-5)'
+    }
+];
+
+const DifficultyPage = () => {
+    return (
+        <main>
+            <title>DCs</title>
+            <h1>Setting DCs</h1>
+            <h2>
+                <Link to="/">Home</Link>
+            </h2>
+            <p></p>
+
+            <h2> Standard Skill DCs </h2>
+            <ul className="listStyles">
```

Comment:
Are we using the same style for all `ul` tags? If so, we should get rid of the `listStyles` class and use the css from that class as the default styling for all `ul`s in the `_defaults.scss` file.

Reply:
Same goes for the `listItemStyles` class

Reply:
The listStyles/listItemStyles were default placeholder stylings kept mainly because it set "max-width."

We can definitely move that into default styling. I'll remove those.